Formula/vg.rb

```diff
@@ -0,0 +1,49 @@
+class Vg < Formula
+  desc "Virtualgo: Easy and powerful workspace based development for go"
+  homepage "https://github.com/GetStream/vg"
+  url "https://github.com/GetStream/vg/archive/v0.8.0.tar.gz"
+  sha256 "e7267b1306c36b3a4a4d10cb9e7ece66a9436a430bbf15028b82313496c92ebe"
+
+  depends_on "go"
+  depends_on "dep"
+  depends_on "bindfs" => :optional
+
+  def install
+    ENV["GOPATH"] = buildpath
+    vgpath = buildpath/"src/github.com/GetStream/vg"
+    vgpath.install buildpath.children
+    cd vgpath do
+      system "dep", "ensure"
+      system "go", "build", "-o", bin/"vg", "-ldflags", "-w -s -X github.com/GetStream/vg/cmd.Version=#{version}"
+      prefix.install_metafiles
+    end
+  end
+
+  def caveats; <<~EOS
```

Comment:
Caveats are generally intended for Homebrew-specific things rather than general usage instructions. I'm not sure any of this applies personally but I'll punt to a maintainer for judgement 🙈.

Reply:
Ok, no problems in removing it though ;)

Reply:
These caveats can all go away.